openshift/template.yaml

```diff
@@ -373,3 +428,70 @@ objects:
     selector:
       matchLabels:
         app: assisted-service
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: assisted-service-envoy-config
+data:
+  main.yaml: |
+    admin:
+      access_log_path: /dev/null
+      address:
+        socket_address: { address: 0.0.0.0, port_value: 9000 }
+
+    static_resources:
+      listeners:
+      - name: ingress
+        address:
+          socket_address: { address: 0.0.0.0, port_value: ${{ENVOY_PORT}} }
+        filter_chains:
+        - filters:
+          - name: envoy.filters.network.http_connection_manager
+            typed_config:
+              "@type": type.googleapis.com/envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager
+              access_log:
+              - name: envoy.access_loggers.file
+                typed_config:
+                  "@type": type.googleapis.com/envoy.extensions.access_loggers.file.v3.FileAccessLog
+                  path: /dev/stdout
+                  jsonFormat:
+                    authority: "%REQ(:AUTHORITY)%"
+                    duration: "%DURATION%"
+                    path: "%REQ(X-ENVOY-ORIGINAL-PATH?:PATH)%"
+                    protocol: "%PROTOCOL%"
+                    response_code: "%RESPONSE_CODE%"
+                    response_flags: "%RESPONSE_FLAGS%"
+                    route_name: "%ROUTE_NAME%"
+                    upstream_host: "%UPSTREAM_HOST%"
```

Comment:
`route_name` and `upstream_host` aren't really interesting here, the route name will always be the same as there is only one route (and it hasn't a name) and the upstream host will also always be `127.0.0.1`.